### PR TITLE
Use ConnectedMusicConfetti for first track upload

### PR DESCRIPTION
**Pull request by sddioulde**
### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?
Locally

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

### CODE REVIEW

File: src/containers/music-confetti/ConnectedMusicConfetti.tsx [hunk 1]
```diff
@@ -12,19 +13,19 @@ const ConnectedMusicConfetti = () => {
...
+  return isVisible ? (
+    <MusicConfetti
+      zIndex={10000}
+      onCompletion={onConfettiFinished}
+      isMatrix={isMatrixMode}
+      limit={isMatrixMode ? (isMobile ? 200 : 500) : undefined}
+      gravity={isMatrixMode ? 0.25 : undefined}
+    />
+  ) : (
+    <></>
```
Comment (raymondjacobson) [inline src/containers/music-confetti/ConnectedMusicConfetti.tsx:28]: could just be null?

File: src/containers/music-confetti/ConnectedMusicConfetti.tsx [hunk 2]
```diff
@@ -12,19 +13,19 @@ const ConnectedMusicConfetti = () => {
   }, [dispatch])
 
   const isVisible = useSelector(getIsVisible)
-  const isMatrix = useSelector(getIsMatrix)
+  const isMatrixMode = isMatrix()
```
Comment (raymondjacobson) [inline src/containers/music-confetti/ConnectedMusicConfetti.tsx:16]: nice

**Review by raymondjacobson — APPROVED** [review]

File: src/containers/music-confetti/ConnectedMusicConfetti.tsx (hunk 1 above)
Comment (sddioulde) [reply src/containers/music-confetti/ConnectedMusicConfetti.tsx:28]: Yeah I guess

Comment (sddioulde) [reply src/containers/music-confetti/ConnectedMusicConfetti.tsx:28]: js client styleguide item to discuss...

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/sd-connected-first-upload-confetti

**Pull request merged**